### PR TITLE
Psu unit tests

### PR DESCRIPTION
**Pull request by andrii-savka**
The implementation of PSU unit tests.

### CODE REVIEW

**Comment by andrii-savka** [comment]
The required updated Jenkins job. 
[QoS test simulation.txt](https://github.com/andrii-savka/sonic-snmpagent/files/1522881/QoS.test.simulation.txt)

File: tests/plugins/psuutil.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+#############################################################################
+
+class PsuUtil():
+    """PSUutil class for SNMP testing"""
+
+    def __init__(self):
+        """ For testing purpose only """
+        self.num_of_psus = 2
+        self.psu1_status = True
+        self.psu2_status = False
```
Comment (qiluo-msft) [inline tests/plugins/psuutil.py:18]: Combine status into a dict #Closed

Comment (andrii-savka) [reply tests/plugins/psuutil.py:18]: Fixed

**Comment by andrii-savka** [comment]
The changes are merged to PR #2

**Pull request closed without merging**